soccer/joystick/GamepadJoystick.hpp

```diff
@@ -8,6 +8,8 @@
  * @brief Logitecch Gamepad/Joystick used to control robots
  *
  * Used to control the Logitech F310 and XBOX controllers
+ *
+ * Appears to have been mostly replaced by GamepadController
```

Comment:
You should be more assertive here, something like "this has been replaced by gamepad controller"